app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -26,6 +26,12 @@
       &.ng-enter-active
         animation-name: bounceInLeft
 
+.CardList-card.is-closed
```

Comment:
I don't think that's correct. It makes sense to have `.Discussion.is-closed`, being that the `Discussion` is on the state closed. But for this scenario I think you should use a modifier like `.CardList.CardList--opaque`, because the fact that the cards inside the list are closed has nothing to do with the list itself. But since you're affecting the appearance of the discussions I believe it's better to go with the first option.